validator/validate_test.go

```diff
@@ -70,6 +71,9 @@ func runSpec(schemas []*gqlparser.Schema, deviations map[string]*Spec, filename
 					if deviation.Errors != nil {
 						spec.Errors = deviation.Errors
 					}
+					if deviation.Skip {
```

Comment:
:joy: I added the same thing in my PR

Reply:
lol, It was a little [MOTTAINAI](https://en.wikipedia.org/wiki/Mottainai)!